src/carousel/carousel.ts

```diff
@@ -54,9 +54,24 @@ export class NgbCarousel implements AfterContentChecked,
   @ContentChildren(NgbSlide) private _slides: QueryList<NgbSlide>;
   private _slideChangeInterval;
 
+  /**
+   *  Amount of time before next slide is shown.
+   */
   @Input() interval = 5000;
+
+  /**
+   *  Whether user can navigate between the first and last slide.
```

Comment:
Whether slide navigation can wrap from the last slide to the first slide.  (or something close)

What you wrote is normal carousel behavior - navigating between first and last slide.  :-)